Student.js

```diff
@@ -188,6 +188,12 @@ function std() {
 }
 
 function res() {
+for(k=0;k<100;k++){
+if(i==200)
+{
+console.logO("Res Data",k)
+}

```

Comment:
Add a closing brace '}' at the end of the function 'res' to ensure proper syntax. [important]